src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -34,6 +35,15 @@
 from WMCore.Services.ReqMgrAux.ReqMgrAux import ReqMgrAux
 
 
+def jobSubmitCondition(jobStats):
+
+    for jobInfo in jobStats:
+        if jobInfo["Current"] >= jobInfo["Threshold"]:
+            return jobInfo["Condition"]
+
+    return "JobSumitReady"
```

Comment:
typo

Reply:
Maybe also add a docstring saying that this method decides whether a job can be submitted or not.